tools/cbmc/include/queue_init.h

```diff
@@ -0,0 +1,129 @@
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "queue_datastructure.h"
+
+/* Using prvCopyDataToQueue together with prvNotifyQueueSetContainer
+   leads to a problem space explosion. Therefore, we use this stub
+   and a sepearted proof on prvCopyDataToQueue to deal with it.
+   As prvNotifyQueueSetContainer is disabled if configUSE_QUEUE_SETS != 1,
+   in other cases the original implementation should be used. */
+#if( configUSE_QUEUE_SETS == 1 )
```

Comment:
I assume the real version changes some set of bytes.  I wonder if you could get most of the effect by havocing a small fix number of randomly selected bytes.  If not, there should be some explanation for why this is sound even tho you're not havocing anything.